### PR TITLE
Update libnice to 0.1.15

### PR DESCRIPTION
**Pull request by colatkinson**
Related to #5798 

I tried getting the build to just use Meson natively, but the documentation was somewhat sparse. So, I've just updated the existing CMake wrapper. I also figured updating GLib would be more onerous, and far more likely to break things.

I also spun up a quick test project to make sure there weren't any missing public headers or objects, which you can check out [here](https://gist.github.com/colatkinson/1fd8b91c79f9494effd99d263896caae).

Let me know if you have any changes you'd like w.r.t. style or functionality.

### CODE REVIEW

File: ports/libnice/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: libnice
 Version: 0.1.13-2
```
Comment (Rastaban) [inline ports/libnice/CONTROL]: Can you update this version number?

Comment (colatkinson) [reply ports/libnice/CONTROL]: Oops, yeah I'll change it later tonight.

Comment (colatkinson) [reply ports/libnice/CONTROL]: Requested changes made

**Comment by Rastaban** [comment]
Awesome, Thanks!

**Pull request merged**